src/repositories/jobs.js

```diff
@@ -16,13 +16,8 @@ const create = async (payload) => {
 
 const getAllJobs = async (query) => {
   try {
-    if (query && query.tags && query.tags.length > 0) {
-      var arr = query.tags.split(",");
-      return await getJobsByTags(arr);
-    } else {
-      const rows = knex("jobs").orderBy("created_at", "desc");
-      return rows;
-    }
+    const rows = knex("jobs").orderBy("created_at", "desc");
```

Comment:
do we also need to get the deleted jobs here? @rohan099 ?

Reply:
@ShubhamJoshi-1994 i think we dont need this api at all coz no one will show all the jobs in his app we need to show jobs by filters

Reply:
ok @rohan099 do we have a function to query based on that? Please mention the scenarios where the user will need to fetch queries

Reply:
@ShubhamJoshi-1994  no we don't have any function but we can cover this in same story where we will search job with filters.